tokio/src/sync/semaphore.rs

```diff
@@ -78,8 +78,8 @@ impl Semaphore {
     pub fn available_permits(&self) -> usize {
         self.ll_sem.available_permits()
     }
-
-    /// Adds `n` new permits to the semaphore.
+    
+    /// The maximum number of permits is usize::MAX >> 3, and this function will panic if the limit is exceeded.
```

Comment:
The "Adds `n` new permits to the semaphore." should stay. Can you also add backticks around `usize::MAX >> 3`?

Reply:
I fixed it now.